cypress/support/pages/setting.page.Object.js

```diff
@@ -0,0 +1,58 @@
+import PageObject from '../PageObject';
+
+class SettingPageObject extends PageObject {
+    get userNameField() {
+        return cy.getByDataCy('userNameField-settings').clear();
+    }
+
+    get emailField() {
+        return cy.getByDataCy('emailField-settings').clear();
+    }
+
+    get passwordField() {
+        return cy.getByDataCy('passwordField-settings').clear();
+    }
+
+    get updateButton() {
+        return cy.getByDataCy('updateButton-settings');
+    }
+
+    get logoutButton() {
+        return cy.get('.btn-outline-danger');
```

Comment:
Consider using a data-cy attribute for the logout button for consistency and to avoid potential issues if the class name changes.